addons/sales_team/migrations/8.0.1.0/post-migration.py

```diff
@@ -20,12 +20,19 @@
 #
 ##############################################################################
 
+import logging
 from openerp import pooler, SUPERUSER_ID
 from openerp.openupgrade import openupgrade, openupgrade_80
 
+logger = logging.getLogger('OpenUpgrade.sales_team')
+
 
 @openupgrade.migrate(no_version=True)
```

Comment:
No. it's too late in crm. crm.case.section moved from crm module (V7.0) to sales_team module (V8.0).

https://github.com/OCA/OpenUpgrade/blob/8.0/addons/sales_team/sales_team.py#L28

kind regards.